model/model.js

```diff
@@ -1,10 +1,19 @@
-const db = require("../db/connection.js")
+const db = require("../db/connection.js");
 
 exports.fetchTopics = () => {
-    return db.query("SELECT * FROM topics").then(({ rows: topics }) => {
-        return topics
+  return db
+    .query("SELECT * FROM topics")
+    .then(({ rows: topics }) => {
+      return topics;
     })
-    .catch((err) => {
-        next(err)
+};
+
+exports.fetchArticleById = (article_id) => {
+  return db.query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then(({ rows: article }) => {
+      if ( article.length === 0){
+        return Promise.reject({ status: 404, msg: "Not Found"})
+      }
+        return article[0]
```

Comment:
I might be tempted to do some array restructuring here, I think that might be clearer than returning article[0]